qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -1042,23 +1042,26 @@ def traces(self, target):
 
     @staticmethod
     def decomp0(target):
-        """Decompose target ~Ud(x, y, z) with 0 uses of the basis gate.
-        Result Ur has trace:
-        :math:`|Tr(Ur.Utarget^dag)| = 4|(cos(x)cos(y)cos(z)+ j sin(x)sin(y)sin(z)|`,
-        which is optimal for all targets and bases"""
+        """Decompose target :math:`~Ud(x, y, z)` with 0 uses of the basis gate.
+        Result  :math:`Ur` has trace:
+
+        .. math::
+            |Tr(Ur.Utarget^dag)| = 4|(cos(x)cos(y)cos(z)+ j sin(x)sin(y)sin(z)|
```

Comment:
```suggestion
        r"""Decompose target :math:`\tilde U_d(x, y, z)` with 0 uses of the basis gate.
        Result  :math:`U_r` has trace:

        .. math::
            \Bigl\lvert \Tr\bigl(U_r U_{\text{target}}^\dagger\bigr)\Bigr\rvert = 4\Bigl\lvert\bigl(\cos(x)\cos(y)\cos(z)+ i \sin(x)\sin(y)\sin(z)\bigr)\Bigr\rvert
```

I think this is what's meant.  Probably needs line breaking, and similar fixes applying to the other methods in this class, if we go for it.